whatrecord/bin/parse.py

```diff
@@ -144,6 +151,9 @@ def parse(
 
     expand : bool, optional
         Expand a substitutions file.
+
+    v3 : bool, optional
+        Use V3 database grammar where applicable.
```

Comment:
Is this referring to EPICS v3?

Reply:
Yes, EPICS V3. Database grammar is a bit different from one to the other, believe it or not...